0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,252 +1,135 @@
 # Interledger Architecture
 
-This document outlines the Interledger architecture and explains how the different layers relate to each other. The [Interledger Protocol (ILP)](#interledger-protocol-ilp) is one layer in the Interledger architecture.
+The Interledger protocol suite provides for secure payments across multiple assets on different ledgers.
 
-The Interledger architecture is heavily inspired by the Internet architecture described in [RFC 1122](https://tools.ietf.org/html/rfc1122), [RFC 1123](https://tools.ietf.org/html/rfc1123) and [RFC 1009](https://tools.ietf.org/html/rfc1009).
+The [Interledger Protocol (ILP)](#interledger-layer) is the core of the Interledger protocol suite.
```

Comment:
I still find our colloquial usage of `ILP` to be unclear.  Let's assume there's some proper name for the collection of all Interledger protocols (the suite).  One of those protocols (the thing in RFC-3) is called “ILP”, or “Interledger Protocol”.  If you put it all together, it reduces to the following: there’s a collection of Interledger protocols, one of which is the "Interledger Protocol.”

Wouldn’t it make more sense to start calling the thing defined by RFC-3 something else, like `ILPC`, which expanded would read `Interledger Protocol Core` or just `Interledger Core Protocol`?

(I'm open to the argument that this naming "ship" has already sailed, but figured I'd give it another try)